modeling-cmds/src/def_enum.rs

```diff
@@ -951,6 +958,14 @@ define_modeling_cmd_enum! {
             pub padding: f32,
         }
 
+        /// Fit the view to the scene with an isometric view.
+        #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, ExecutionPlanFromMemory, ModelingCmdVariantEmpty)]
+        pub struct ViewIsometric {
+            /// How much to pad the view frame by.
```

Comment:
What unit is the padding?

Reply:
this already merged and is in main,  see my comment about the diff being confusing due to me having to base this off of earlier release + cherrypicks.   my diff is only for the face center stuff.